android_beta/8.0beta.yml

```diff
@@ -287,8 +292,100 @@ notes:
 
   - note: >
       Cross-device and cross-app import is now limited to Thunderbird, as it was intended for a
-      way to migrate K-9 and Thunderbir Desktop users to Thunderbird for Android.
+      way to migrate K-9 and Thunderbird Desktop users to Thunderbird for Android.
     short_note: Cross-device import limited to Thunderbird
     k9mail_only: true
     tag: fixed
     group: 4
+
+  ## Beta 5
+  # New
+  - note: >
+      The unread message colors have been updated to make it more apparent when a message is
+      unread.
+    short_note: Unread message colors are updated for better visibility.
+    tag: new
+    group: 5
+
+  - note: >
+      We're now including a prompt to encourage users to make financial contributions. On beta,
+      the timing of the prompt is lower than it will be on release, so you have a chance to
+      test it.
+    short_note: >
+      You will be asked for a financial contribution once you've used the app for a while. Help
+      us continue active development on Thunderbird for Android!
+    thunderbird_only: true
+    tag: new
+    group: 5
+
+  - note: >
+      Added a thank you message to the funding experience, making it more obvious we are
+      thankful.
+    tag: new
+    group: 5
+
+  - note: >
+      When using the QR code import, encourage users to upgrade Thunderbird Desktop where the
+      feature has just landed.
+    short_note: QR code import now suggests upgrading Thunderbird Desktop.
+    tag: new
+    group: 5
+
+  - note: >
+      A big thank you to @aindriu80 who took the Irish translation to 100% very quickly. Irish
+      is now available to all users.
+    short_note: Irish translation included and 100% complete
+    tag: new
+    group: 5
+
+
+  # Fixed
+  - note: >
+      Folder drawer width is limited to the Material drawer width, improving tablet layout.
+    short_note: Folder drawer width adjusted for tablet layouts
+    tag: fixed
+    group: 5
+
+  - note: >
+      Non-system languages are available again in the language switcher.
```

Comment:
The languages were always present, selecting them just didn't change the app language.